src/main/java/org/geppetto/simulation/manager/GeppettoManager.java

```diff
@@ -918,4 +925,13 @@ public Path downloadProject(IGeppettoProject project) throws GeppettoExecutionEx
 
 		return zip;
 	}
+	
+	public boolean getAllowVolatileProjectsSimulation() {
```

Comment:
These methods will go away and move inside GeppettoManagerConfiguration.

Reply:
went away